content/en/real_user_monitoring/browser/monitoring_vitals_performance.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Monitoring Vitals Performance
+kind: documentation
+further_reading:
+  - link: "https://learn.datadoghq.com/courses/core-web-vitals-lab"
+    tag: "Learning Center"
+    text: "Interactive Lab: Core Web Vitals"
+  - link: "https://www.datadoghq.com/blog/real-user-monitoring-with-datadog/"
+    tag: "Blog"
+    text: "Real User Monitoring"
+  - link: "https://www.datadoghq.com/blog/core-web-vitals-monitoring-datadog-rum-synthetics/"
+    tag: "Blog"
+    text: "Monitor Core Web Vitals with Datadog RUM and Synthetic Monitoring"
+---
+
+{{< beta-callout url="https://www.google.com" >}}
+Try the beta! Vitals are in public beta.
+{{< /beta-callout >}} 
+
+## Overview
+
+{{< img src="real_user_monitoring/browser/vitals_performance/vitals-workflow.mp4" alt="RUM Vitals helps you find the root cause of browser performance issues based on real user traffic." video="true" >}}
+
+RUM Vitals helps you find the true root cause of browser performance issues based on real user traffic. You can troubleshoot browser metrics, including the [Core Web Vitals][1] (CWV), [Loading Time][2] (Datadog's custom metric that evaluates how long it takes for a page to fully load from the user's perspective), and more. Determine what caused a slow page, with all the relevant information in one place.
+
+## Prerequisites
+
+To get the most out of this feature, Datadog recommends that you use:
+
+- [RUM Browser SDK][3] version 5.4.0 or newer
+- [Session Replay][4] for at least some sessions
+
+## Select a vital
+
+Navigate to the [Vitals page][5], or click the "Vitals" tab under the [**Digital Experience > Performance Monitoring**][6] page.
+
+{{< img src="real_user_monitoring/browser/vitals_performance/vitals-page-selectors-1.png" alt="You can check vitals for the top most visited pages or specific pages." style="width:100%;" >}}
+
+From this view, you have two ways of selecting the page and vital to review and optimize:
+
+- Select from a list of the most visited pages
+- Enter a view name on the left and select the page
+
+## Filter and evaluate
+
+After you've selected a page and vital, you can see insights that explain their performance.
+
+From here, you can:
+
+- Change the time frame in the upper right
+- Filter by different session attributes using the dropdown buttons
```

Comment:
can we just call them "attributes" instead of "session attributes"? technically these are attributes of the view event, not the session